amethyst_renderer/src/config.rs

```diff
@@ -45,10 +45,18 @@ impl Default for Config {
         }
     }
 }
-fn default_title() -> String { "Amethyst game".to_string() }
-fn default_vsync() -> bool { true }
-fn default_multisampling() -> u16 { 1 }
-fn default_visibility() -> bool { true }
+fn default_title() -> String {
```

Comment:
Can we use `#[cfg_attr(rustfmt, rustfmt_skip)]` for these default functions?  I like the old formatting better.

Reply:
I'm fine with either.

Reply:
I'd prefer to avoid skipping for consistency

Reply:
I guess I can live with that.

Reply:
We can make it default to have single expression functions on a single line if you want.

Reply:
That works!

Reply:
Can I fix that as a separate PR after all the current ones are merged? Getting messed up in the head from all the PRs..

Reply:
Sure